flaskshop/settings.py

```diff
@@ -2,14 +2,16 @@
 """Application configuration."""
 import os
 from pathlib import Path
+from dotenv import load_dotenv
 
+load_dotenv()
 class LocalConfig:
     db_type = os.getenv('DB_TYPE', 'mysql')
     user = os.getenv('DB_USER', 'root')
-    passwd = os.getenv('DB_PASSWD', '123456')
```

Comment:
use black

Reply:
I need update all files with black, and check everything with flake8, but it is in another branch, because we have a lot of changes